openedx_lti_tool_plugin/models.py

```diff
@@ -208,31 +208,49 @@ def __str__(self) -> str:
         return f'<LtiProfile, ID: {self.id}>'
 
 
-class CourseAccessConfiguration(models.Model):
-    """Course access configuration.
+class LtiToolConfiguration(models.Model):
+    """LTI Tool Configuration.
 
-    A model to store LTI tool course access configuration.
+    A model to store LTI tool configuration.
     """
 
     EXAMPLE_ID_LIST = 'Example: ["id-1", "id-2", ...]'
 
+    NEW_ACCOUNTS_ONLY = 'new_only'
+    EXISTING_AND_NEW = 'existing_and_new'
+    EXISTING_ONLY = 'existing_only'
+
+    USER_PROVISIONING_MODE_CHOICES = [
+        (NEW_ACCOUNTS_ONLY, _('New accounts only.')),
+        (EXISTING_AND_NEW, _('Existing and new accounts.')),
+        (EXISTING_ONLY, _('Existing accounts only.')),
```

Comment:
```suggestion
        (NEW_ACCOUNTS_ONLY, _('New accounts only (automatic)')),
        (EXISTING_AND_NEW, _('Existing and new accounts (prompt)')),
        (EXISTING_ONLY, _('Existing accounts only (prompt)')),
```
@Serafin-dev What do you think about adding a description to know if the provisioning mode is automatic or prompts the user to login?